docs/content/users/usage/developer-tools.md

```diff
@@ -113,7 +113,7 @@ On Drupal 8+, if you want to use `drush uli` on the host (or other Drush command
 
 ## DDEV and Terminus
 
-[Terminus](https://pantheon.io/docs/guides/terminus) is a command line tool providing advanced interaction with [Pantheon](https://pantheon.io/) services. `terminus` is available inside the project’s container, allowing users to get information from, or issue commands to their Pantheon-hosted sites. This is an especially helpful feature for Windows users since Terminus is only officially supported on Unix-based systems.
+[Terminus](https://pantheon.io/docs/terminus) is a command line tool providing advanced interaction with [Pantheon](https://pantheon.io/) services. `terminus` is available inside the project’s container, allowing users to get information from, or issue commands to their Pantheon-hosted sites. This is an especially helpful feature for Windows users since Terminus is only officially supported on Unix-based systems.
```

Comment:
Interesting, this was recently changed from the docs/terminus URL when they started failing on it...

Reply:
Wait a minute, I think it just changed again.  Now it's redirecting to https://docs.pantheon.io/terminus.  Updated.

Reply:
What a squirrelly link!

Reply:
I do think we should be able to convince the link checker to follow redirects.

Reply:
Direct links are best for the reader and redirects risk going stale or turning into long chains. I think flagging redirects and keeping links current is best, even with the inconvenience of outages and weird cases like this.